### ISSUE TITLE
Do not overwrite existing executables in /usr/local/bin

### ISSUE THREAD

**Issue by spockz** [kitematic/kitematic#362]
When installing docker through homebrew it deploys the executables to /usr/local/bin. Opening Kitematic overwrites theses files without warning or asking. This is quite inconvenient if you already have some specific version installed for compatibility reasons.

Please do not overwrite files and ask whether the executables should be installed and warn if it would overwrite any files.

**Comment by jeffdm** [comment]
Thanks for the feedback!

**Comment by robbles** [comment]
+1 on this. Please fix this soon!

This is a pretty terrible default - there's no indication that anything has been magically installed in system directories just by opening the app. I've been using boot2docker, and after opening Kitematic for the first time, my previously working install spits out this error for `docker ps`:

```
FATA[0000] Error response from daemon: client and server don't have same version (client : 1.17, server: 1.15)
```

**Comment by jeffdm** [comment]
@robbles shoot, sorry this happened. Labeled the issue as a bug and working to fix this.

**Comment by TeckniX** [comment]
Same issue here, was using boot2docker and wanted to checkout the Beta of Kitematic - I'm now in the same boat as everyone else with a 'bad/old' server version:
`client and server don't have same version (client : 1.18, server: 1.16)`

**Comment by jeffdm** [comment]
@TeckniX sorry about that. Running `docker-machine upgrade dev` on the command line should work (make sure the VM is running with `docker-machine start dev` first.

Working on a fix to make Kitematic self-contained.

**Comment by TeckniX** [comment]
Thanks Jeff for the tip - this worked for me :+1:

**Comment by jeffdm** [comment]
Fixed in Kitematic 0.7. Closing
